firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java

```diff
@@ -296,27 +324,71 @@ AppDistributionReleaseInternal getCachedLatestRelease() {
         AlertDialog.BUTTON_POSITIVE,
         context.getString(R.string.update_yes_button),
         (dialogInterface, i) ->
-            // show download progress in notification manager
+        // show download progress in notification manager
```

Comment:
nit: this comment is explaining why updateApp is called with `true`, so we should probably move it to right above line 330